### PR TITLE
Drag and drop file update

### PR DESCRIPTION
**Pull request by jomaora**
**Detailed purpose of the PR**

Update views and properties on the drag and drop component

**Result and observation**

**Default**
Capture d’écran 2020-03-05 à 11 45 03

**Drag file**
Capture d’écran 2020-03-05 à 11 48 12

**Loading**
Capture d’écran 2020-03-05 à 11 50 09

**Error**
Capture d’écran 2020-03-05 à 11 50 00

**Success**
Capture d’écran 2020-03-05 à 11 50 19

**Ready**
Capture d’écran 2020-03-05 à 14 27 21

**Testing Strategy**

- Manual testing
- Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/common.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 }
 
 .box {
-  width: 360px;
+  width: 436px;
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/common.css]: imho every component should be flex, and the parent is responsible to set the width (eventually depending on media queries)

**Comment by jomaora** [comment]
[Update scrom drag-and-drop component](https://trello.com/c/us8Xw2Wr/1636-update-scrom-drag-and-drop-component)

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/index.js [hunk 2]
```diff
@@ -11,49 +11,65 @@ import {PlaceHolder} from './placeholder';
...
 
-  const canDisplayPreview = previewContent && previewContent.src && previewContent.type;
+  const loading = type === 'loading';
 
   return (
     <Dropzone key={idBox} accept={accept} disabled={loading} onDrop={onDrop}>
       {({getRootProps, getInputProps, isDragActive}) => {
         const getView = () => {
-          if (errorMessage || canDisplayPreview) {
+          if (type === 'success' || type === 'error' || type === 'ready') {
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/index.js]:
```suggestion
          if (includes(type, ['success', 'error', 'ready'])) {
```

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/preview.js [hunk 3]
```diff
@@ -1,30 +0,0 @@
...
-  const translate = get('translate', context);
-  switch (type) {
-    case 'image':
-      return <img src={src} />;
-
-    case 'video':
-      return <video controls width="100%" src={src} type="video/*" />;
-
-    case 'scorm':
-      return <p>{translate('scorm')}</p>;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/preview.js]: supprime la clef de trad

File: packages/@coorpacademy-components/src/molecule/drag-and-drop/preview.js [hunk 4]
```diff
@@ -1,30 +0,0 @@
...
-    default:
-      return <p>{translate('no_preview_available')}</p>;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/drag-and-drop/preview.js]: supprime la clef de trad

**Pull request merged**